homepage/docs/index.md

```diff
@@ -6,5 +6,6 @@ implementations for FIDO Device Onboard specification.
 Following snapshots are available for different releases of FIDO Device Onboard implementations.
 
 * [latest](https://secure-device-onboard.github.io/docs-fidoiot/latest)
+* [1.1.1](https://secure-device-onboard.github.io/docs-fidoiot/1.1.1)
```

Comment:
Can you update https://secure-device-onboard.github.io/docs-fidoiot/latest/releases/ also ?

Reply:
yes